### ISSUE TITLE
Separate Beckn Gateway and Beckn Registry installations 

### ISSUE THREAD

**Issue by vbabuEM** [beckn/beckn-utilities#88]
This issue has to be approved by business @faizmagic 

Right now Beckn Gateway and Beckn Registry are clubbed together in the v2 install script. Is this really a production setup. Wouldnt it be better if it is separate options.

Please start on this only after @faizmagic  has approved this enhancement.

**Comment by faizmagic** [comment]
@vbabuEM , i suppose this already done as part of the March 19th event. can you add the necessary details and  close the ticket as required
